<?xml version="1.0" encoding="UTF-8"?><testrun name="TestEnablers" project="it.unibo.radarSystem22" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testsuite name="it.unibo.radarSystem22.sprint3.TestEnablers" time="15.206">
    <testcase name="testEnablers" classname="it.unibo.radarSystem22.sprint3.TestEnablers" time="15.206">
      <error>java.lang.NullPointerException&#13;
	at it.unibo.radarSystem22.sprint2a.proxy.LedProxyAsClient.getState(LedProxyAsClient.java:34)&#13;
	at it.unibo.radarSystem22.sprint3.TestEnablers.testEnablers(TestEnablers.java:89)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)&#13;
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)&#13;
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:274)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:48)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:242)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:240)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:48)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:233)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:303)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:89)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:542)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:770)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:464)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)&#13;
&#13;
</error>
    </testcase>
  </testsuite>
</testrun>
